### PR TITLE
Update config API generator and generated files for 1.24

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
LGTM

I wouldn't be confident checking the Golang part of the changes.

File: genref/output/md/kubeadm-config.v1beta3.md [hunk 1]
```diff
@@ -264,109 +264,6 @@ node only (e.g. the node ip).</p>
   
     
 
-## `BootstrapToken`     {#BootstrapToken}
-
```
Comment (kbhawkey) [inline genref/output/md/kubeadm-config.v1beta3.md:268]: :wave: @tengqm Are `BootstrapToken` and `BootstrapTokenString` replaced by similar content later in the file?
It is difficult to identify the individual changes.

Comment (tengqm) [reply genref/output/md/kubeadm-config.v1beta3.md:268]: These are generated contents, I didn't verify the changes line by line.
As for the content, both `BootstrapToken` and `BootstrapTokenString` look consistent.

**Pull request merged**